frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -447,6 +457,19 @@ const HostDetailsPage = ({
     }
   );
 
+  const {
+    data: hostCertificates,
+    isLoading: isLoadingHostCertificates,
+    isError: isErrorHostCertificates,
+  } = useQuery(
+    ["hostCertificates", host_id],
+    () => hostAPI.getHostCertificates(hostIdFromURL),
```

Comment:
Do you want to include the [params](https://github.com/fleetdm/fleet/pull/26457/files) for pagination and sort here and in the device user page as well?